node-module-system/add.ts

```diff
@@ -0,0 +1,5 @@
+export { add };
```

Comment:
Please use `export function add` syntax. This syntax is really ugly and hard to use.